plugins/woocommerce-admin/client/products/use-product-helper.ts

```diff
@@ -82,25 +82,23 @@ export function useProductHelper() {
 				status,
 			} ).then(
 				( newProduct ) => {
+					const noticeContent =
+						newProduct.status === 'publish'
+							? __(
+									'Product published. View in store.',
+									'woocommerce'
+							  )
+							: __(
+									'Product successfully created.',
+									'woocommerce'
+							  );
 					if ( ! skipNotice ) {
-						createNotice(
-							'success',
-							newProduct.status === 'publish'
-								? __(
-										'🎉 Product published. View in store',
-										'woocommerce'
-								  )
-								: __(
-										'🎉 Product successfully created.',
-										'woocommerce'
-								  ),
-							{
-								actions: getNoticePreviewActions(
-									newProduct.status,
-									newProduct.permalink
-								),
-							}
-						);
+						createNotice( 'success', `🎉‎ ${ noticeContent }`, {
```

Comment:
Github does not show the unicode character here which is `‎[U+200E]` and should be represented like `🎉‎[U+200E] ${ noticeContent }`.